public/404.html

```diff
@@ -41,7 +41,7 @@
           </a>
         </div>
         <div class="govuk-header__content">
-          <a href="/additional-payments/landing-page" class="govuk-header__link govuk-header__service-name"
+          <a href="#" class="govuk-header__link govuk-header__service-name"
```

Comment:
Should we remove this a tag? or change it to a `https://www.gov.uk/government/collections/additional-payments-for-teaching-eligibility-and-payment-details`?

Reply:
I think it needs an anchor tag for the styling, but we can address this in a follow up.